cypress/support/index.js

```diff
@@ -19,7 +19,7 @@
 // ***********************************************************
 
 // Import commands.js using ES2015 syntax:
-import '../utils/commands';
+import { currentBackendEndpoint } from '../utils/commands';
```

Comment:
Can we add one more import section for import `currentBackendEndpoint`? It's looks a little bit strange. Not sure if it can be work with tree shrinking.

Reply:
Sure, the code has been updated.